### PR TITLE
Create quick-start-btpanel.md

### PR DESCRIPTION
**Pull request by bestlaw66**
在国内有大部分用户都在使用宝塔面板管理服务器，因此增加使用宝塔面板部署的教程，可视化的部署文档可以帮助用户更加便捷的部署apollo

## What's the purpose of this PR

XXXXX

## Which issue(s) this PR fixes:
Fixes #

## Brief changelog

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [ ] Read the [Contributing Guide](https://github.com/apolloconfig/apollo/blob/master/CONTRIBUTING.md) before making this pull request.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit tests to verify the code.
- [ ] Run `mvn clean test` to make sure this pull request doesn't break anything.
- [ ] Update the [`CHANGES` log](https://github.com/apolloconfig/apollo/blob/master/CHANGES.md).

## Summary by CodeRabbit

- **New Features**
	- Added deployment guidance for Apollo using aaPanel and BTPanel (available in English and Chinese).
	- Introduced new sidebar entries for quick access to the deployment guides.

- **Documentation**
	- Detailed step-by-step instructions for deploying Apollo with Docker on aaPanel and BTPanel, including prerequisites and configuration settings.
	- Included security considerations regarding port exposure and firewall configuration in the deployment guides.

### CODE REVIEW

**Comment by nobodyiam** [comment]
请参考 #4283 增加 sidebar 链接以及英文说明

**Comment by bestlaw66** [comment]
> 请参考 #4283 增加 sidebar 链接以及英文说明

已参考更新

File: docs/en/deployment/third-party-tool-btpanel.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+To install aaPanel, go to the [aaPanel](https://www.aapanel.com/new/download.html#install) official website and select the corresponding script to download and install.
+
+## Deployment
+
+aaPanel(Applicable versions 7.0.11 and above) Deployment guidelines
+
+1. Log in to aaPanel and click `Docker` in the menu bar
+
+   ![Docker](/docs/en/images/deployment/btpanel/install.png)
```
[the remark this answers is not in the document]
Comment (bestlaw66) [reply docs/en/deployment/third-party-tool-btpanel.md:19]: 相对路径不适用此场景

**Comment by nobodyiam** [comment]
You may run `docsify serve docs` to check if the layout and images display correctly, as they don't render properly on my end.

image

**Comment by nobodyiam** [comment]
BTW, please sign the CLA as instructed by the CLA bot.

**Comment by bestlaw66** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/en/deployment/third-party-tool-btpanel.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+To install aaPanel, go to the [aaPanel](https://www.aapanel.com/new/download.html#install) official website and select the corresponding script to download and install.
+
+## Deployment
+
+aaPanel(Applicable versions 7.0.11 and above) Deployment guidelines
+
+1. Log in to aaPanel and click `Docker` in the menu bar
+
+   ![Docker](en/images/deployment/btpanel/install.png)
```
Comment (nobodyiam) [inline docs/en/deployment/third-party-tool-btpanel.md:17]: Does it really work in your environment? It doesn't work for me.

image

I suppose it should be fixed like this

```suggestion
   ![Docker](../images/deployment/btpanel/install.png)
```

**Comment by bestlaw66** [comment]
image
image
I can now confirm that the image can be displayed normally.

**Comment by nobodyiam** [comment]
I believe you can do better.

image

**Comment by bestlaw66** [comment]
> I believe you can do better.
> 
> image

Sorry, I overlooked this point, but now I have corrected it.

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**